assets/store/User.js

```diff
@@ -25,6 +25,7 @@ export default class User extends Reactive {
     this.prop('rw', 'highlightKeywords', []);
     this.prop('rw', 'rtc', {});
     this.prop('rw', 'status', 'pending');
+    this.prop('rw', 'ignoreStatuses', 1);
```

Comment:
I think the default should be changed from `1`to `false`. Also, changing this from "rw" to "persist" (no need for `key` here) will make the browser remember the setting.